extras/linked-token/README.md

```diff
@@ -1,66 +1,200 @@
-## Foundry
+# Linked Token Readme
 
-**Foundry is a blazing fast, portable and modular toolkit for Ethereum application development written in Rust.**
+## Outline for Linked Token Bridge
+- Contracts
+    - ERC20 token named USDCTest on Calibration Network
+    - Token Controller contract on Calibration Network
+    - Token Replica contract on subnet
+- Workflow
+    - Deposit tokens from rootnet to subnet
+        - calling the rootBridge.bridgeToken() to lock up USDC tokens and initiate an IPC cross net message to the subnet token bridge
+        - subnet token bridge receives IPC message and mints USDC proxy tokens
+    - Withdraw tokens from subnet to rootnet
+        - call subnetBridge.withdrawToken() to burn proxy USDC tokens on the subnet and initiate an IPC cross net message to the root network bridge
 
-Foundry consists of:
+## USDC propagation flow
 
-- **Forge**: Ethereum testing framework (like Truffle, Hardhat and DappTools).
-- **Cast**: Swiss army knife for interacting with EVM smart contracts, sending transactions and getting chain data.
-- **Anvil**: Local Ethereum node, akin to Ganache, Hardhat Network.
-- **Chisel**: Fast, utilitarian, and verbose solidity REPL.
+A high-level overview of the process is shown in the following diagrams:
 
-## Documentation
+![Linked Token Bridging](./linked-token-bridging.png)
 
-https://book.getfoundry.sh/
+![Linked Token Withdrawing](./linked-token-withdrawing.png)
 
-## Usage
+## Setup and instructions for running on calibnet
 
-### Build
+1. Follow IPC guide for setting up a subnet pinned to the filecoin calibnet network: https://github.com/consensus-shipyard/ipc/blob/main/docs/ipc/quickstart-calibration.md
+2. copy the config file from ipc/contracts/script/examples/cross-token/.env.template to .en a config file for connecting to calibnet and your subnet. Create a new wallet with for executing the USDC test that is not one of the validators. You will need to fund this wallet using the calibnet facuet. Set PRIVATE_KEY in .env for this new wallet.
+3. Deploy or use a pre-deployed gateway on calibnet
```

Comment:
We should prefer the pre-deployed path, whereas the instructions below ask for redeployment.